src/measure_theory/bochner_integration.lean

```diff
@@ -76,6 +76,8 @@ local infixr ` →ₛ `:25 := simple_func
 namespace simple_func
 
 section bintegral
+/-! ### Define the Bochner integral of simple functions of the type `α →ₛ β` where `β` is a normed
```

Comment:
In case it helps, you can preview how the docstrings will render using [this link](https://observablehq.com/@bryangingechen/github-lean-doc-preview?url=%221757%22).

Reply:
Oh, thanks. That is very helpful.